copyfromstaging.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+TMP=$(mktemp -d)
+
+copy_one_api() {
+  PACKAGE=$1
+  STAGING_DIR=owl-bot-staging/$1
+  PACKAGE_DIR=apis/$1
+
+  # All source files need the extension .g.cs.  Some will only be generated with the
+  # extension .cs.  Rename files that end only in .cs.
+  for cs_file in $(find "$STAGING_DIR" -name '*.cs'); do
+    if [[ $cs_file != *.g.cs ]]; then
+      mv $cs_file $(echo $cs_file | sed -E 's/.cs$/.g.cs/g')
+    fi
+  done
+
+  # Remove the newly generated standalone snippets until they are ready for surfacing.
+  rm -rf "$STAGING_DIR/$1.StandaloneSnippets"
+
+  # We generate our own project files
+  rm -f $(find "$STAGING_DIR" -name '*.csproj')
```

Comment:
Except I don't think we do at the moment... we should be running `generateprojects.sh` (in the root directory) as part of this script, unless we expect that nothing within the project structure would change due to Owlbot... which is entirely feasible, of course. Will need to discuss that with @amanda-tarafa.

Reply:
We should copy the chunk of code that generates the .csproj files from generateprojects.sh into this script.

Reply:
We not just run generateprojects.sh? It basically runs some dotnet code. The entire meaningful content is:

```sh
dotnet run -p tools/Google.Cloud.Tools.ReleaseManager generate-projects
```

While we *could* copy that directly into this file, we've then got an extra place to change if we rename ReleaseManager. It feels like running the shell script is simpler.

(However, it may be unnecessary, as I mentioned before - because I don't expect OwlBot to change anything in terms of what would be generated.)

Reply:
> We not just run generateprojects.sh? It basically runs some dotnet code. The entire meaningful content is:
> 
> ```shell
> dotnet run -p tools/Google.Cloud.Tools.ReleaseManager generate-projects
> ```
> 
> While we _could_ copy that directly into this file, we've then got an extra place to change if we rename ReleaseManager. It feels like running the shell script is simpler.
> 
> (However, it may be unnecessary, as I mentioned before - because I don't expect OwlBot to change anything in terms of what would be generated.)

Indeed, we can just run it.